seaworthy/pytest/fixtures.py

```diff
@@ -18,13 +18,18 @@ def docker_helper_fixture(name='docker_helper', scope='module'):
 
         docker_helper = docker_helper_fixture(scope='class')
 
+    :param name: The name of the fixture.
+    :param scope: The scope of the fixture.
+    :param kwargs:
+        Keyword arguments to pass to the ``DockerHelper`` constructor.
     """
     @pytest.fixture(name=name, scope=scope)
     def fixture():
-        prefix = 'test'
+        namespace = kwargs.pop('namespace', 'test')
         if 'PYTEST_XDIST_WORKER' in os.environ:
```

Comment:
Stick `# pragma: no cover` on the end of this line to exclude the xdist branch from coverage. We don't really care about that at the moment.